library/ssl_tls13_client.c

```diff
@@ -2719,6 +2720,7 @@ static int ssl_encrypted_extensions_parse( mbedtls_ssl_context* ssl,
 
```

Comment:
Trivia:
* Spacing issue in L.2709. 
* Non-uniform way of calling the extension parsers between ALPN, MFL and EarlyData. Suggest the way used for EarlyData: `ret = ext(); if( ret != 0 ) { ... }`. 
* Non-uniform error reporting when the parsers fail: We should either always add a `MBEDTLS_SSL_DEBUG_RET` or nowhere, and always use the exact name of the function that has been called, here `ssl_parse_encrypted_extensions_early_data_ext`.

Once `ext_size` is a `size_t`, the explicit cast is no longer necessary here.